changelog/13228.feature.rst

```diff
@@ -0,0 +1 @@
+``pytest.HIDDEN_PARAM`` can now be used in ``id`` of ``pytest.param`` or in ``ids`` of ``parametrize``. It hides the parameter set from the test name.
```

Comment:
Change `pytest.HIDDEN_PARAM` to a link to the `Constants` section in the "API Reference" (see my other comment).

Reply:
I haven't managed to find any mentions of `NotChecked` in #13192.
I've copied the naming from `NOTSET`, which is also used around parameter set naming, although not public.
And they both follow [PEP8 style for constants](https://peps.python.org/pep-0008/#constants).